near/src/index.tsx

```diff
@@ -25,22 +25,22 @@ async function initConnection() {
   // Needed to access wallet
   const wallet = new WalletConnection(near, null);
 
-  const api = await initNear(wallet.account(), { contractId: 'filecoin-bridge.testnet' });
+  await requestSignIn(wallet);
 
-  return { api, wallet }
-}
-
-initConnection()
-  .then(({ api, wallet }) => {
-    ReactDOM.render(
-      <App
-        api={api as any}
-        wallet={wallet}
-      />,
-      document.getElementById('root')
-    );
+  const api = await init(wallet.account(), {
```

Comment:
these can actually be left out, but I'm include them here just for demo purposes